traj/src/traj/param_max_reachable_vel.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+"""
+To calculates the maximum reachable velocity at the end of the segment based on the position difference (p_start and p_end) and the starting velocity (v_start), 
+taking into considereation vel/acc/jrk constraints. this idea is the same idea behind the TOPP-RA paper: "A New Approach to Time-Optimal Path Parameterization
+based on Reachability Analysis [H. pham 2018]", where they calculate max reachable velocity based on only the acc constraint, Here, we added the jerk constraint as well to calcuate 
+the max reachable velocity at the end of the segment
+
+to be done:
+to add zero velocity whenever there is change in direction, then it works for positive and negative velocity as well 
+
+paper link: https://www.researchgate.net/publication/318671280_A_New_Approach_to_Time-Optimal_Path_Parameterization_Based_on_Reachability_Analysis
+"""
+
+import traj
+import numpy as np
+import math
+    
+##### find idx for stop points of path, points at which velocity direction changes   
```

Comment:
Consider making these comments docstrings instead:
```python
def set_stp_pts_to_zero(path):
    """
    Find idx for stop points of path, points at which velocity direction changes.
    """
    .....
```
That is the pythonic way to document functions, and if you type `set_stp_pts_to_zero?` in ipython or `help(set_stp_pts_to_zero)` in the standard python shell, it'll print the docstring.